ios/Classes/DRM/FLTEzdrmAssetsLoaderDelegate.m

```diff
@@ -0,0 +1,113 @@
+//
+//  FLTEzdrmAssetsLoaderDelegate.m
+//  better_player
+//
+//  Created by Koldo on 18/05/2021.
+//
+
+#import "FLTEzdrmAssetsLoaderDelegate.h"
+
+@implementation FLTEzdrmAssetsLoaderDelegate
+
+NSString *_assetId;
+
+NSString * KEY_SERVER_URL = @"https://fps.ezdrm.com/api/licenses/";
+
+- (instancetype)initWithCertificateUrl:(NSURL *)certificateUrl {
+    self = [super init];
+    _certificateURL = certificateUrl;
+    return self;
+}
+
+- (void)removeKeyWith:(NSFileManager *)fileManager {
+
+}
+
+/*------------------------------------------
+**
+** getContentKeyAndLeaseExpiryfromKeyServerModuleWithRequest
+**
+** takes the bundled SPC and sends it the the
+** key server defined at KEY_SERVER_URL in the View Controller
+** it returns a CKC which then is returned.
+** ---------------------------------------*/
+- (NSData *)getContentKeyAndLeaseExpiryfromKeyServerModuleWithRequest:(NSData*)requestBytes and:(NSString *)assetId and:(NSString *)customParams and:(NSError *)errorOut {
+    NSData * decodedData;
+    NSURLResponse * response;
+    NSURL * ksmURL = [[NSURL alloc] initWithString: [NSString stringWithFormat:@"%@%@%@",KEY_SERVER_URL,assetId,customParams]];
+
+    NSMutableURLRequest * request = [[NSMutableURLRequest alloc] initWithURL:ksmURL];
+    [request setHTTPMethod:@"POST"];
+    [request setValue:@"application/octet-stream" forHTTPHeaderField:@"Content-type"];
+    [request setHTTPBody:requestBytes];
+
+    @try {
+        decodedData = [NSURLConnection sendSynchronousRequest:request returningResponse:&response error:nil];
+    }
+    @catch (NSException* excp) {
+        NSLog(@"SDK Error, SDK responded with Error: (error)");
+    }
+    return decodedData;
+}
+
+/*------------------------------------------
+ **
+ ** getAppCertificate
+ **
+ ** returns the apps certificate for authenticating against your server
+ ** the example here uses a local certificate
+ ** but you may need to edit this function to point to your certificate
+ ** ---------------------------------------*/
+- (NSData *)getAppCertificate:(NSString *) String {
+    NSData * certificate = nil;
+    certificate = [NSData dataWithContentsOfURL:_certificateURL];
+    return certificate;
+}
+
+- (BOOL)resourceLoader:(AVAssetResourceLoader *)resourceLoader shouldWaitForLoadingOfRequestedResource:(AVAssetResourceLoadingRequest *)loadingRequest {
+    NSURL *assetURI = loadingRequest.request.URL;
+    NSString * str = assetURI.absoluteString;
+    NSString * mySubstring = [str substringFromIndex:str.length - 36];
```

Comment:
`str` and `mySubstring` aren't best way to describe what this string does. Can you provide some better names?. Also str.length - 36 - are you sure that it will be always 36?